test/spmd/test_xla_sharding.py

```diff
@@ -630,8 +635,8 @@ def test_shard_hashing(self):
     self.assertTrue(torch.allclose(xt1 + 0, xt2 + 0))
 
     # Check that hashes are different for the sharded and non-sharded tensors
-    hash1 = torch_xla._XLAC._get_graph_hash([xt1])
-    hash2 = torch_xla._XLAC._get_graph_hash([xt2])
+    hash1 = torch_xla._XLAC._get_graph_hash([xt1 + 0])
```

Comment:
Why?

Reply:
`xt1` is already a device data, `_get_graph_hash ` will call `CollectSyncTensors` and it will return an empty list(since it think `xt1` don't need to be synced). This test should not use `_get_graph_hash ` as it is an internal API but I am a bit lazy to rewrite the test. 

Forcing it to be `xt1 + 0` will let `CollectSyncTensors` use the interedmate tensor as root and calculate the hash.